apiary.apib

```diff
@@ -8600,7 +8600,8 @@ Creates a new workspace for an existing configuration in [Development Branch](#r
     + loginType: (optional, enum[string]) - Request credentials type for new workspace. backend attribute is required when loginType is set.
         + Members
             + default
```

Comment:
ten default by tady asi nemel byt.

a toto ti chybi u https://connection.keboola.com/v2/storage/workspaces?async={async}' (vyrabeni WS bez konfigurace)

Reply:
default ten endpoint podporuje, je tam na to aj test

Reply:
https://github.com/keboola/connection/blob/e9f10e3f97a55eda50a06cbb1604f55434020ce3/src/Manage/StorageBackend/StorageBackendInternalLoginType.php#L45-L47

Reply:
aha, ok. jen pls doplnis to i do toho druheho endpointu ?

Reply:
Jo přidal sem to všude a přidal sem tam backtick protože to apiary špatně vykreslovalo

Reply:
pak su happy